components/application_manager/opapp.h

```diff
@@ -0,0 +1,82 @@
+/**
+ * ORB Software. Copyright (c) 2022 Ocean Blue Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * App model
+ *
+ * Note: This file is part of the platform-agnostic application manager library.
+ */
+
+#ifndef HBBTV_SERVICE_OPAPP_H
```

Comment:
BLOCKING - This guard is named wrong - it should just be the name of the file in caps and underscores.